package.json

```diff
@@ -221,9 +247,24 @@
         ],
         "dependencies": [
           "@babel/core",
+          "@typescript-eslint/eslint-plugin",
```

Comment:
applied version group for packages that are defined as `dependencies` for packages that are being published to npm registry

this fixes syncpack error, not sure if this is the right way to go:

> https://github.com/microsoft/fluentui/pull/18808#issuecomment-872827200